README.md

```diff
@@ -1,51 +1,94 @@
-# <u>refine.bio Examples Guide: </u>
-This repository contains example workflows of how to use data downloaded from 
+# __refine.bio Examples Guide:__
+This repository contains example workflows of how to use data downloaded from
 [refine.bio](https://www.refine.bio).
 
+## How to use:
+
+If you are comfortable with GitHub, we recommend [cloning](https://help.github.com/articles/cloning-a-repository/)
+this repository and following the examples.
+However, if you are less comfortable with GitHub, you can also use the green
+button that says "Clone or download" to "Download ZIP" and then use these
+notebooks as you please on your own computer.
+Make sure you are on the master branch.
```

Comment:
If a person is less comfortable with Github, I think they are unlikely to be on a branch other than master and may find this confusing.